docs/dev/daily-workflows.rst

```diff
@@ -703,13 +703,19 @@ Making a new release
   line, e.g., using ``set noeol`` in Vim.  This is also configured in
   ``./editorconfig``)
 
-* Run ``LOCKSET=tools/towncrier ./py -m towncrier`` to auto-generate the changelog.
+* Run ``LOCKSET=towncrier/3.12.6 ./py -m towncrier`` to auto-generate the changelog.
 
   - You may append ``--draft`` to see a preview of the changelog update without
     actually modifying the filesystem.
 
   - (WIP: `lablup/backend.ai#427 <https://github.com/lablup/backend.ai/pull/427>`_).
 
+  - Alternatively, you can use the following command to automatically fetch the Python interpreter version
+
+  ```bash
+  LOCKSET=towncrier/$(yq '.python.interpreter_constraints[0] | split("==") | .[1]' pants.toml) ./py -m towncrier
+  ```
```

Comment:
Unfortunately, this is not Markdown... 😢

The preview:
<img width="866" alt="image" src="https://github.com/user-attachments/assets/bff51211-0024-4177-82c7-dd3da57aa3e4">

We should use (with a caution to indentation matched with the item list):
```suggestion
    .. code-block:: shell
    
       LOCKSET=towncrier/$(yq '.python.interpreter_constraints[0] | split("==") | .[1]' pants.toml) ./py -m towncrier
```

Reply:
Ah, I mistook it for markdown, thanks for the confirmation.